.gitignore

```diff
@@ -14,4 +14,86 @@ auth/keycloak/config/hsperfdata_jboss/
 auth/keycloak/config/*.log
 trufflehog_report.*
 datastore.xml
-**/.vscode/
\ No newline at end of file
+**/.vscode/
+*.bak
+database/mssql/scripts/dbscripts/PSP_PIMS_S23_01/._Alter Down
```

Comment:
@dfilteau can you explain what is going on here? also, If you need to ignore files I'd suggest using wildcards instead of creating one row per file.

Reply:
Pr looks good besides this.